examples/2016-10-31/image_resize_python/README.md

```diff
@@ -0,0 +1,44 @@
+# AWS::Serverless::LayerVersion Library Code Example
+
+This example shows you how to download and publish an application that bundles its Function dependencies into an AWS::Serverless::LayerVersion resource.
```

Comment:
Can also add explanation like "The function resizes an image in the source bucket and saves the output to the target bucket" to give a bit more context. I could only find out what the example does after clicking the link at the bottom.

Reply:
Good call, updated